packages/react-aria-components/docs/GridList.mdx

```diff
@@ -159,12 +160,14 @@ import {GridList, Item, Button} from 'react-aria-components';
   @supports selector(:has(.foo)) {
     gap: 0;
 
-    & .react-aria-Item[aria-selected=true]:has(+ [aria-selected=true]) {
+    .react-aria-Item[aria-selected=true]:has(+ [aria-selected=true]),
+    .react-aria-Item[aria-selected=true]:has(+ .react-aria-DropIndicator + [aria-selected=true]) {
```

Comment:
I guess we should probably change the drop indicator color in this case as well so you can see it but can do that later.